### PR TITLE
network buffering / mlat stagger customisation option

### PR DESCRIPTION
**Pull request by wiedehopf**
Increasing the beast-reduce flush interval will reduce network load for aggregators by sending less but larger network packets.

mlat-client startup can take quite some time when using several aggregators, add an ENV var so people can reduce it if they prefer.

mlathub was using beast_reduce_out to send data, this is local transfer so using beast_reduce is not necessary. (mlat results are not high message rate in any case)

### CODE REVIEW

**Comment by dirkhh** [comment]
mlat-client startup time is a question I do get not infrequently when people are looking at the logs. I think I'll experiment with reducing that in the adsb.im feeder image from the default 15 seconds

**Comment by rkolb-cmt** [comment]
The 15 seconds was chosen to allow `mlat-client` to start up and connect to a server, and produce all relevant log messages for this, before starting up the next instance of `mlat-client`. This avoids having `mlat-client` logging messages interlacing, which makes the logs very hard to follow. Yes, it's a compromise, but since restarts of Ultrafeeder should be infrequent, a slow deployment of the MLAT connections shouldn't really be a big deal.

In any case, the proposed change is NOT changing the default value but simply allows the user to change the `mlat-client` startup stagger time if they wish so.

**Review by rkolb-cmt — APPROVED** [review]

**Review by kx1t — APPROVED** [review]
